phys2bids/reporting/html_report.py

```diff
@@ -0,0 +1,219 @@
+"""Reporting functionality for phys2bids."""
+import sys
+from distutils.dir_util import copy_tree
+from os.path import join as opj
```

Comment:
This is quite different from how we import `os` in any other script in the package. I can't find it now, but we changed it in the first place because it was more "standard" not to use aliases for `os`.
Could you please change it to:

```suggestion
from os.path import join
```
Or even:
```suggestion
import os
```

instead?
It's to be consistent across scripts, that's all.